cmake/modules/StandaloneOverlay.cmake

```diff
@@ -32,11 +32,13 @@ list(APPEND CMAKE_MODULE_PATH
   ${LLVM_CMAKE_MODULES_PATH}
 )
 
-message(STATUS "FINDING LIPO")
-  execute_process(
-  COMMAND "xcrun" "-find" "lipo"
-  OUTPUT_VARIABLE LIPO
-  OUTPUT_STRIP_TRAILING_WHITESPACE)
+set(SWIFT_DARWIN_XCRUN_TOOLCHAIN "XcodeDefault" CACHE STRING
```

Comment:
Also I dislike how this assumes Darwin.

Reply:
I agree, that it is less than ideal that this assume Darwin.  I don't know how to handle that really.